app/bundles/LeadBundle/EventListener/DashboardSubscriber.php

```diff
@@ -492,5 +498,77 @@ public function onWidgetDetailGenerate(WidgetDetailEvent $event): void
 
             return;
         }
+
+        if ('number.contacts' == $event->getType()) {
+            $numContacts = $this->leadModel->getNumberContacts();
+            $subtitle    = $this->translator->trans('mautic.widget.number.contacts.description');
```

Comment:
As mentioned in the twig template, it's simpler to translate strings there. The less there is to read the faster developers can understand the code.
```suggestion
            $subtitle    = 'mautic.widget.number.contacts.description';
```